Source/AwsCommonRuntimeKit/sdkutils/CRTAWSProfileProperty.swift

```diff
@@ -6,17 +6,19 @@ import AwsCSdkUtils
 public struct CRTAWSProfileProperty {
     let rawValue: OpaquePointer
 
-    public init(rawValue: OpaquePointer) {
+    init(rawValue: OpaquePointer) {
         self.rawValue = rawValue
     }
 
+    /// Returns the property's string value
     public var value: String? {
```

Comment:
I think this can be ~String?~ -> `String`?

The [comments](https://github.com/awslabs/aws-c-sdkutils/blob/4068c2c1f1342a79f8a42aeed848c7efabc1161f/include/aws/sdkutils/aws_profile.h#L117-L121) say:
```
/**
 * Returns a reference to the property's string value
 */
AWS_SDKUTILS_API
const struct aws_string *aws_profile_property_get_value(const struct aws_profile_property *property);
```

which *sounds* like it's always there.

And looking a the source code ... `aws_profile_property->value` is never NULL: https://github.com/awslabs/aws-c-sdkutils/blob/4068c2c1f1342a79f8a42aeed848c7efabc1161f/source/aws_profile.c#L256

Reply:
Thanks, fixed.